tests/sles4sap/wizard_hana_install.pm

```diff
@@ -7,45 +7,92 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Install HANA with SAP Installation Wizard
+# Summary: Install HANA with SAP Installation Wizard. Verify installation with
+# sles4sap/hana_test
 # Maintainer: Ricardo Branco <rbranco@suse.de>
 
 use base 'sles4sap';
 use strict;
 use testapi;
-use utils 'turn_off_gnome_screensaver';
-use utils qw(type_string_slow zypper_call);
+use utils qw(type_string_slow zypper_call turn_off_gnome_screensaver);
+use ipmi_backend_utils 'use_ssh_serial_console';
 use version_utils 'is_sle';
 
 sub get_total_mem {
-    if (check_var('BACKEND', 'qemu')) {
-        return get_required_var('QEMURAM');
+    return get_required_var('QEMURAM') if (check_var('BACKEND', 'qemu'));
+    my $mem = script_output q@grep ^MemTotal /proc/meminfo | awk '{print $2}'@;
+    $mem /= 1024;
```

Comment:
So you set mem to 1GB if you can't detect the value, right? Is it not better to die in that case?

Reply:
No, I divide whatever is in `/proc/meminfo` by 1024 to have the memory in the same units as QEMURAM. Values in `/proc/meminfo` are in Kb, while QEMURAM are in MiB. You're thinking of `//=` which is "assign if undef"; `/=` is an arithmetic division unitary operator.